admin_scripts/os2displaypc/chromium_install.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+# Minimal install of X and Chromium and connectivity.
+apt-get install -y xinit xserver-xorg-core x11-xserver-utils --no-install-recommends --no-install-suggests
+apt-get install -y  xdg-utils xserver-xorg-video-qxl xserver-xorg-video-fbdev xserver-xorg-video-vesa xserver-xorg-video
```

Comment:
Do I understand correctly that `xserver-xorg-video-qxl` is only useful for virtualisation environments? If so, then I'd remove it.

Actually, I'd just lean towards installing `xserver-xorg-video-all` to give us the best possible hardware support. (If that doesn't drag in too many other random packages...?)

Reply:
Good point! I do want to have the `qxl` support, in all cases, because I develop and test on virtual environments all the time.

Reply:
Changed to  `xserver-xorg-video-all`.